docs/modules/intellij-idea-plugin/pages/user-guide.adoc

```diff
@@ -0,0 +1,82 @@
+= Payara IntelliJ IDEA Plugin User Guide
+:man-prefix: #
+:man-suffix: 1
+:imagesdir: images/
+
+== Introduction
+
+The Payara Ultimate Tools plugin enables integration of Payara Cloud and Platform with IntelliJ IDEA Ultimate Edition. The Payara Tools plugin is available in the https://plugins.jetbrains.com/plugin/15114-payara-platform-tools[JetBrains Plugins Marketplace].
```

Comment:
Is it Payara Tools plugin for IntelliJ IDEA Ultimate? The name Payara Ultimate Tools plugin can be confusing to the reader.